src/Application/Cli/PreviewCommand.php

```diff
@@ -237,7 +237,7 @@ private function isFound(string $executablePath): bool
 
     private function fileListToDisplay(array $files, string $sourceDirectory): string
     {
-        $files = array_map(function (string $file) use ($sourceDirectory): string {
+        $files = array_map(static function (string $file) use ($sourceDirectory): string {
```

Comment:
Same for these, they are not useful and add noise.

Reply:
It's a suggestion from PHP Inspections EA:
> [EA] This closure can be declared as static (better scoping; in some cases can improve performance).

Please tell me if you really want to remove them, I will do it.